pnpm devをするとuv_signal_start EINVAL -> ポート5173に何も描画されない

```
~/WebstormProjects/misskey $ pnpm dev

> misskey@2024.5.0 dev /home/kisaragi/WebstormProjects/misskey
> node scripts/dev.mjs


> misskey@2024.5.0 clean /home/kisaragi/WebstormProjects/misskey
> node ./scripts/clean.js


> misskey@2024.5.0 build-pre /home/kisaragi/WebstormProjects/misskey
> node ./scripts/build-pre.js


> misskey@2024.5.0 build-assets /home/kisaragi/WebstormProjects/misskey
> node ./scripts/build-assets.mjs


> misskey-js@2024.5.0 build /home/kisaragi/WebstormProjects/misskey/packages/misskey-js
> node ./build.js

(node:51218) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
[misskey-js] start building...
[misskey-js] build succeeded.
[misskey-js] finish building.

> misskey-bubble-game@0.0.1 build /home/kisaragi/WebstormProjects/misskey/packages/misskey-bubble-game
> node ./build.js


> misskey-reversi@0.0.1 build /home/kisaragi/WebstormProjects/misskey/packages/misskey-reversi
> node ./build.js

[misskey-bubble-game] start building...
[misskey-bubble-game] build succeeded.
[misskey-reversi] start building...
[misskey-reversi] build succeeded.
[misskey-reversi] finish building.
[misskey-bubble-game] finish building.

> misskey@2024.5.0 build-pre /home/kisaragi/WebstormProjects/misskey
> node ./scripts/build-pre.js "--watch"


> misskey@2024.5.0 build-assets /home/kisaragi/WebstormProjects/misskey
> node ./scripts/build-assets.mjs "--watch"


> misskey-bubble-game@0.0.1 watch /home/kisaragi/WebstormProjects/misskey/packages/misskey-bubble-game
> nodemon -w package.json -e json --exec "node ./build.js --watch"


> sw@ watch /home/kisaragi/WebstormProjects/misskey/packages/sw
> nodemon -w ../../package.json -e json --exec "node build.js watch"


> misskey-reversi@0.0.1 watch /home/kisaragi/WebstormProjects/misskey/packages/misskey-reversi
> nodemon -w package.json -e json --exec "node ./build.js --watch"

(node:51595) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

> misskey-js@2024.5.0 watch /home/kisaragi/WebstormProjects/misskey/packages/misskey-js
> nodemon -w package.json -e json --exec "node ./build.js --watch"


> frontend@ dev /home/kisaragi/WebstormProjects/misskey/packages/frontend
> vite --config vite.config.local-dev.ts --debug hmr


> backend@ dev /home/kisaragi/WebstormProjects/misskey/packages/backend
> node ./scripts/dev.mjs

[nodemon] 3.1.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): ../../package.json
[nodemon] watching extensions: json
[nodemon] starting `node build.js watch`
[nodemon] 3.0.2
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): package.json
[nodemon] watching extensions: json
[nodemon] starting `node ./build.js --watch`
[nodemon] 3.1.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): package.json
[nodemon] watching extensions: json
[nodemon] starting `node ./build.js --watch`
[nodemon] 3.0.2
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): package.json
[nodemon] watching extensions: json
[nodemon] starting `node ./build.js --watch`
(node:51710) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
[misskey-reversi] start watching...
[misskey-bubble-game] start watching...
[misskey-bubble-game] detect changed...
[misskey-reversi] detect changed...
[misskey-js] start watching...
[misskey-js] detect changed...
[nodemon] 3.1.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): src/**/*
[nodemon] watching extensions: ts,js,mjs,cjs,json
[nodemon] starting `pnpm run build`
Starting SW building...
watching...
Re-optimizing dependencies because lockfile has changed

> backend@ build /home/kisaragi/WebstormProjects/misskey/packages/backend
> swc src -d built -D --strip-leading-paths


  VITE v5.3.2  ready in 485 ms

  ➜  Local:   http://127.0.0.1:5173/
Successfully compiled: 824 files, copied 27 files with swc (535.01ms)
[nodemon] clean exit - waiting for changes before restart

> misskey@2024.5.0 build-assets /home/kisaragi/WebstormProjects/misskey
> node ./scripts/build-assets.mjs

(node:52093) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
[misskey-reversi] finish watching.
node:internal/errors:563
    ErrorCaptureStackTrace(err);
    ^

Error: uv_signal_start EINVAL
    at process.startListeningIfSignal (node:internal/process/signal:36:13)
    at process.emit (node:events:514:28)
    at _addListener (node:events:557:14)
    at process.addListener (node:events:606:10)
    at file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-reversi/build.js:98:11
    at new Promise (<anonymous>)
    at watchSrc (file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-reversi/build.js:94:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-reversi/build.js:30:2 {
  errno: -22,
  code: 'EINVAL',
  syscall: 'uv_signal_start'
}

Node.js v20.10.0
[nodemon] app crashed - waiting for file changes before starting...
[misskey-bubble-game] finish watching.
node:internal/errors:563
    ErrorCaptureStackTrace(err);
    ^

Error: uv_signal_start EINVAL
    at process.startListeningIfSignal (node:internal/process/signal:36:13)
    at process.emit (node:events:514:28)
    at _addListener (node:events:557:14)
    at process.addListener (node:events:606:10)
    at file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-bubble-game/build.js:98:11
    at new Promise (<anonymous>)
    at watchSrc (file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-bubble-game/build.js:94:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-bubble-game/build.js:30:2 {
  errno: -22,
  code: 'EINVAL',
  syscall: 'uv_signal_start'
}

Node.js v20.10.0
[nodemon] app crashed - waiting for file changes before starting...
[misskey-js] finish watching.
node:internal/errors:563
    ErrorCaptureStackTrace(err);
    ^

Error: uv_signal_start EINVAL
    at process.startListeningIfSignal (node:internal/process/signal:36:13)
    at process.emit (node:events:514:28)
    at _addListener (node:events:557:14)
    at process.addListener (node:events:606:10)
    at file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-js/build.js:98:11
    at new Promise (<anonymous>)
    at watchSrc (file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-js/build.js:94:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///home/kisaragi/WebstormProjects/misskey/packages/misskey-js/build.js:30:2 {
  errno: -22,
  code: 'EINVAL',
  syscall: 'uv_signal_start'
}

Node.js v20.10.0
[nodemon] app crashed - waiting for file changes before starting...
  _____ _         _           
 |     |_|___ ___| |_ ___ _ _ 
 | | | | |_ -|_ -| '_| -_| | |
 |_|_|_|_|___|___|_,_|___|_  |
 v2024.5.0               |___|

 Misskey is an open-source decentralized microblogging platform.
 If you like Misskey, please donate to support development. https://www.patreon.com/syuilo

--- debian12-gen13 (PID: 52126) ---
INFO *  [core boot]     Welcome to Misskey!
INFO *  [core boot]     Misskey v2024.5.0
INFO *  [core boot env] NODE_ENV: development
WARN *  [core boot env] The environment is not in production mode.
WARN *  [core boot env] DO NOT USE FOR PRODUCTION PURPOSE!
VERB *  [core boot machine]     Hostname: debian12-gen13
VERB *  [core boot machine]     Platform: linux Arch: x64
VERB *  [core boot machine]     CPU: 24 core MEM: 31.1GB (available: 18.2GB)
INFO *  [core boot nodejs]      Version v20.10.0 detected.
DONE *  [core boot config]      Loaded
DONE *  [core boot]     Misskey initialized
INFO *  [core nest]     NestFactory: Starting Nest application...
INFO *  [core nest]     InstanceLoader: MainModule dependencies initialized
INFO *  [core nest]     InstanceLoader: QueueModule dependencies initialized
ERR  *  [core nest]     AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1667:5): 
INFO *  [core]  The process is going to exit with code 1
file:///home/kisaragi/WebstormProjects/misskey/node_modules/.pnpm/execa@9.2.0/node_modules/execa/lib/return/final-error.js:6
        return new ErrorClass(message, options);
               ^

ExecaError: Command failed with exit code 1: ./built/boot/entry.js
    at getFinalError (file:///home/kisaragi/WebstormProjects/misskey/node_modules/.pnpm/execa@9.2.0/node_modules/execa/lib/return/final-error.js:6:9)
    at makeError (file:///home/kisaragi/WebstormProjects/misskey/node_modules/.pnpm/execa@9.2.0/node_modules/execa/lib/return/result.js:108:16)
    at getAsyncResult (file:///home/kisaragi/WebstormProjects/misskey/node_modules/.pnpm/execa@9.2.0/node_modules/execa/lib/methods/main-async.js:174:4)
    at handlePromise (file:///home/kisaragi/WebstormProjects/misskey/node_modules/.pnpm/execa@9.2.0/node_modules/execa/lib/methods/main-async.js:157:17) {
  shortMessage: 'Command failed with exit code 1: ./built/boot/entry.js',
  command: './built/boot/entry.js',
  escapedCommand: './built/boot/entry.js',
  cwd: '/home/kisaragi/WebstormProjects/misskey/packages/backend',
  durationMs: 1805.173455,
  failed: true,
  timedOut: false,
  isCanceled: false,
  isGracefullyCanceled: false,
  isTerminated: false,
  isMaxBuffer: false,
  isForcefullyTerminated: false,
  exitCode: 1,
  stdio: [ undefined, undefined, undefined, undefined ],
  ipcOutput: [],
  pipedFrom: []
}

Node.js v20.10.0
/home/kisaragi/WebstormProjects/misskey/packages/backend:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  backend@ dev: `node ./scripts/dev.mjs`
Exit status 1
```

備考: [既知](https://carrienonadise.hatenablog.com/entry/2024/05/19/222623)

--- a/packages/misskey-bubble-game/build.js
+++ b/packages/misskey-bubble-game/build.js
@@ -95,7 +95,6 @@ async function watchSrc() {
 		process.on('SIGHUP', resolve);
 		process.on('SIGINT', resolve);
 		process.on('SIGTERM', resolve);
-		process.on('SIGKILL', resolve);
 		process.on('uncaughtException', reject);
 		process.on('exit', resolve);
 	}).finally(async () => {

--- a/packages/misskey-js/build.js
+++ b/packages/misskey-js/build.js
@@ -95,7 +95,6 @@ async function watchSrc() {
 		process.on('SIGHUP', resolve);
 		process.on('SIGINT', resolve);
 		process.on('SIGTERM', resolve);
-		process.on('SIGKILL', resolve);
 		process.on('uncaughtException', reject);
 		process.on('exit', resolve);
 	}).finally(async () => {

--- a/packages/misskey-reversi/build.js
+++ b/packages/misskey-reversi/build.js
@@ -95,7 +95,6 @@ async function watchSrc() {
 		process.on('SIGHUP', resolve);
 		process.on('SIGINT', resolve);
 		process.on('SIGTERM', resolve);
-		process.on('SIGKILL', resolve);
 		process.on('uncaughtException', reject);
 		process.on('exit', resolve);
 	}).finally(async () => {

